### PR TITLE
feat(electric): Add support for TIME data type in electrified tables

### PR DESCRIPTION
**Pull request by alco**
Closes VAX-857.

### CODE REVIEW

**Comment by alco** [comment]
@magnetised Looping you in for this PR just to see if you maybe have a different take on the implementation from Ilia's.

File: .changeset/shy-days-whisper.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@core/electric": patch
```
Comment (magnetised) [inline .changeset/shy-days-whisper.md:2]: shouldn't this also include the client?

File: components/electric/lib/electric/satellite/serialization.ex [hunk 2]
```diff
@@ -511,6 +512,25 @@ defmodule Electric.Satellite.Serialization do
     val
   end
 
+  def decode_column_value!(val, :time) do
```
Comment (magnetised) [inline components/electric/lib/electric/satellite/serialization.ex:515]: this serialisation module is getting biiig. might be worth splitting the column value support into a separate module

**Review by magnetised — APPROVED** [review]
this looks absolutely fine to me. great stuff

File: .changeset/shy-days-whisper.md (hunk 1 above)
Comment (alco) [reply .changeset/shy-days-whisper.md:2]: The only change done on the client is to not crash when receiving a value of type TIME. All the user-facing changes are being implemented in DAL by Kevin.

File: components/electric/lib/electric/satellite/serialization.ex (hunk 2 above)
Comment (alco) [reply components/electric/lib/electric/satellite/serialization.ex:515]: True! Good point. I'll refactor it after merging this and the Bytea utility module from https://github.com/electric-sql/electric/pull/386 into main.

**Pull request merged**